### ISSUE TITLE
Markdown plugin not parsing class links correctly

### ISSUE THREAD

**Issue by Rhonun** [tgreyuk/typedoc-plugin-markdown#9]
According to http://typedoc.org/guides/doccomments/ in the second paragraph _Additionally you can link to other classes, members or functions using double square brackets._

E.g. you link to another class not with the {@link OtherClass} markdown tag but with [[OtherClass]]. This plugin doesn't parse that and you are left with [[OtherClass]] on the .md file

**Comment by tgreyuk** [comment]
Hello. 

I have mocked up a page for this and it does appear to render correctly with my tests (https://github.com/tgreyuk/typedoc-plugin-markdown/blob/master/tests/mocks/default/modules/_doc_comments_.md).

Could I please confirm that you are using the latest version of the plugin? If you are using the latest version then please let me know what version of TypeDoc you are using with any additional useful info so I can replicate the issue.

Many thanks.

**Comment by Rhonun** [comment]
So I was using the [[foo]] notation described in the typedoc documentation becasue {@link} was not working... however I noticed that you were using [foo](bar) notation. This doesn't autolink to the default theme or this one


For reference I was doing 

```/**
 * @name AbstractMetadataModule
 * @description
 * Provides the module for the [[AbstractMetadataComponent]]
 * ```json
 * {
 *  declarations: [AbstractMetadataComponent],
 *  exports: [AbstractMetadataComponent],
 *  imports: [
 *    IonicModule,
 *    StackIonSelectModule,
 *  ],
 *  providers: [],
 * }
 * ```
 */```

Which links to the correct class in the default theme

I looked at the way you did it with the []()

```
/**
 * @name AbstractMetadataModule
 * @description
 * Provides the module for the [AbstractMetadataComponent](./abstract_metadata_component.ts)
 * ```json
 * {
 *  declarations: [AbstractMetadataComponent],
 *  exports: [AbstractMetadataComponent],
 *  imports: [
 *    IonicModule,
 *    StackIonSelectModule,
 *  ],
 *  providers: [],
 * }
 * ```
 */``` 

And it's saying it can't find the file because for the default theme it's `abstractmetadatacomponent.html` and for this plugin it's `abstractmetadatamodule.md`

Could we support the [[]] notation that the default theme uses?

**Comment by tgreyuk** [comment]
Hi.. yes the link i sent you was generated using [[]] notation. Here is the source that is was generated from - https://github.com/tgreyuk/typedoc-plugin-markdown/blob/master/tests/src/doc-comments.ts#L9. I will see if I can replicate the issues you are having somehow. Just to confirm you are using latest version of the plugin?

**Comment by Rhonun** [comment]
My package.json says  `"typedoc-plugin-markdown": "^1.0.7",`

Here's the full file where the [[]] are not working.

Could it be because i'm getting the AbstractMetadataComponent from a barrell file?

```import {NgModule} from '@angular/core';

import {AbstractMetadataComponent} from './index';
import {IonicModule} from 'ionic-angular';
import {StackIonSelectModule} from '../../../../../shared/components/stack_ion_select/stack_ion_select.component.module';

/**
 * @name AbstractMetadataModule
 * @description
 * Provides the module for the [[AbstractMetadataComponent]]
 * ```json
 * {
 *  declarations: [AbstractMetadataComponent],
 *  exports: [AbstractMetadataComponent],
 *  imports: [
 *    IonicModule,
 *    StackIonSelectModule,
 *  ],
 *  providers: [],
 * }
 * ```
 */
@NgModule({
  declarations: [AbstractMetadataComponent],
  exports: [AbstractMetadataComponent],
  imports: [
    IonicModule,
    StackIonSelectModule,
  ],
  providers: [],
})
export class AbstractMetadataModule {
}
```

**Comment by tgreyuk** [comment]
The issue was that the comments inside the @description tag were not being parsed properly.

The issue should now be fixed in new version (v1.0.8).

Many Thanks

**Comment by Rhonun** [comment]
Can confirm! Thanks for the quick turnaround 💯
